recipes/beyla/collector-config.yaml

```diff
@@ -24,26 +24,64 @@ spec:
         protocols:
           grpc:
           http:
-
+    connectors:
+      spanmetrics/httpserver:
+        histogram:
+          unit: 's'
+          explicit:
+            buckets: [10us, 50us, 100us, 200us, 400us, 800us, 1ms, 2ms, 4ms, 6ms, 8ms, 10ms, 50ms, 100ms, 200ms, 400ms, 800ms, 1s, 1400ms, 2s, 5s, 10s, 15s]
+        dimensions:
+          - name: http.method
+            default: GET
+          - name: http.status_code
+          - name: net.host.name
+          - name: http.route
+        exclude_dimensions: ['status.code', 'span.kind', 'span.name', 'service.name']
+        namespace: http.server
     processors:
+      filter/httpserveronly:
+        error_mode: ignore
+        traces:
+          span:
+            - attributes["http.method"] == nil
+            - kind.string != "Server"
+            - attributes["net.host.name"] == "127.0.0.1"
       resourcedetection:
         detectors: [env, gcp]
         timeout: 2s
         override: false
-
+      groupbyattrs:
+        keys:
+          - net.host.name
+      resource:
+        attributes:
+          - key: k8s.pod.ip
+            from_attribute: net.host.name
+            action: insert
+      k8sattributes:
+        pod_association:
+          - sources:
+            - from: resource_attribute
+              name: k8s.pod.ip
+      filter/podonly:
```

Comment:
consider adding a comment to explain why this is needed

Reply:
added comments generally